packages/modules/src/utils/Types.ts

```diff
@@ -19,7 +20,7 @@ export interface SessionKeyManagerModuleConfig extends BaseValidationModuleConfi
   version?: ModuleVersion;
   nodeClientUrl?: string;
   smartAccountAddress: string;
-  storageType?: StorageType;
+  sessionStorageClient?: ISessionStorage;
```

Comment:
can rename this to customSessionStorageClient

Reply:
Fixed